src/CodeGenerator/src/Generator/PaginationGenerator.php

```diff
@@ -114,7 +114,7 @@ private function generateOutputPagination(Operation $operation, Pagination $pagi
             [$returnType, $iteratorType] = $this->typeGenerator->getPhpType($listShape);
             $iteratorTypes[] = $iteratorType;
 
-            $getter = 'get' . $resultKey;
+            $getter = 'get' . \ucfirst($resultKey);
```

Comment:
this could be moved up, and be used in line 104 as well

Reply:
good catch, thanks